package.json

```diff
@@ -51,10 +51,10 @@
     "prepublishOnly": "turbo run build",
     "lint-staged": "lint-staged",
     "next-with-deps": "./scripts/next-with-deps.sh",
-    "next": "cross-env NEXT_PRIVATE_SKIP_CANARY_CHECK=1 NEXT_TELEMETRY_DISABLED=1 NODE_OPTIONS=\"--trace-deprecation --enable-source-maps\" next",
+    "next": "cross-env NEXT_NEW_DEV_OVERLAY=1 NEXT_PRIVATE_SKIP_CANARY_CHECK=1 NEXT_TELEMETRY_DISABLED=1 NODE_OPTIONS=\"--trace-deprecation --enable-source-maps\" next",
```

Comment:
I think this is a bit early. We should do this once we're close to feature parity.